src/test/java/seedu/application/logic/commands/AddCommandTest.java

```diff
@@ -190,6 +215,11 @@ public void addApplication(Application application) {
         public ReadOnlyApplicationBook getApplicationBook() {
             return new ApplicationBook();
         }
+
+        @Override
+        public void commitApplicationBook() {
+
```

Comment:
I think it would be good to leave a comment to explain that this is intentionally left empty
```suggestion
            // Do nothing
```